Oshanimi/SigninToScoring.feature

```diff
@@ -0,0 +1,146 @@
+	| Email or phone textfield | beatriceoluwakemisola@gmail.com| Next   |
+	| Enter your password      | VirusDetectedK006				| Next   | 
+   And user is at the homepage
+    
+		Examples: 
+		| Name of browsers |
+		| Chrome           |
+		| Edge             |
+
+Scenario Outline: 1.1_09 Sign in with 'forgets email address' option
+	Given user has 'Choose an option' dialog box displayed on the screen
+	When user clicks on 'Use another account'button
+	And user clicks on 'Forget email?' button
+	And  user has 'Find your email' dialog box displayed  
+	Then 'the following Features are displayed on the screen' simultaneaously
+		| Features                            | Action																| Button |
+		| Enter	                              | Valid input															| Click  |
+		| 'Email or phone' text field         | beatriceoluwakemisola@gmail.com										| Next   |
+		| What's your name? dialog box        | Enter First name 'Beatrice' and Last name 'O' name in the textfield | Next   |
+		| Get a verification code dialog box  |																		| Send   |
+		| Enter the code  dialog box          | Enter verification code '631025'									| Next   |
+   And 'choose an account' dialog box pops up
+   And user selects the 'google account' button
+   And enters the scoring homepage
+ 
+Scenario: 1.1_10 Sign in with a valid password in the 'email or phone number' text field
+	Given user has 'Choose an option' dialog box displayed on the screen
+	And user has a valid google account
+	And user clicks on 'Use another account' button
+	When user fills the account 'Email or phone' text field with value 'VirusDetectedK006'
+	And user clicks the 'Next' button
+	Then the 'Email textfield'is marked red. 
+	And a text 'Couldn't find your Google account' appears in the validation errors region
+
+Scenario Outline: 1.1_11 Sign in with valid email and unplugs internet connection 
+	Given user has 'Choose an option' dialog box displayed on the screen
+	And user clicks on 'Use another account' button
+	When user fills the account 'Email or phone'  textbox with value 'correct@gmail.com'
+	And user turnoff internet connection 
+	And user clicks on 'Next' button
+    Then display a dialogue box 'Something went wrong' 
+	And 'Sorry, something went wrong there. 
+	And Try again.' dialog box appears
+
+Scenario Outline: 1.1_12 Sign in with valid email and the laptop sleeps due to inactivity 
+	but user fails to click 'Next' button
+	Given  user has 'Choose an option' dialog box displayed on the screen
+	And user clicks on 'Use another account' button
+	When user fills the account 'Email or phone' text field with value 'incorrect@gmail.com'
+	And user fails to click the 'Next' button 
+    And user screen goes off due to inactivity
+	And user hits the mouse
+	Then user's screen resumed
+	And display a dialogue box 'You're not signed in' 
+	And User has 'Your session ended because there was no activity' displayed on the screen
+	And 'Try again' button in the dialog box
+
```

Comment:
Thanks for you job! 
1) Some scenarios will be better to combine into Outlines, e.g. when we check valid/invalid phone number, email, empty password input and etc.
2) you provided a great test coverage
3) it will be better to revise some info from the lesson (about syntax of steps)
4) please learn how to format document in Visual Studio